addons/l10n_sa_edi/__manifest__.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Saudi Arabia - E-invoicing',
+    'icon': '/l10n_sa/static/description/icon.png',
+    'version': '0.1',
+    'depends': [
+        'base',
+        'account_edi_ubl',
+        'account_debit_note',
```

Comment:
@mebe-odoo It is not about them supporting, it is whether we need to support it.  (is there a case where someone has no other option but to create a debit note? )

Reply:
No such cases, no. It is optional, so I suppose we could make it an accessory module ?

Reply:
I had a discussion with Nada Seder (sen) concerning the debit_note dependency for the localization module. Given that support for the issuance of debit notes for ZATCA is as integral as the issuance of credit notes, whether for simplified or standard invoices, it would be prudent to either:

1. Keep the dependency
2. Create a bridge module for debit notes

Personally, I believe creating the bridge module would be a cleaner approach, but let me know if that is something you agree with before we move forward with the change.

Reply:
@mebe-odoo We need to be certain that the debit notes are needed in the sense that there is a legal requirement for most of the clients.  Then it can just be kept in here, no need to split it.